src/webots/vrml/WbNode.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include "WbNode.hpp"
 
+#include "../nodes/utils/WbNodeUtilities.hpp"
```

Comment:
Unfortunately this is not good because it will cause a circular dependency between `nodes/utils` and `vrml` module.
Sorry but I did see this issue when suggesting to move the functionality.

Other alternatives would be to remove this dependency by implementing the functionality to search for hidden fields in WbNode class or to move the `removeInvisibleProtoNodes` to the WbBaseNode class that is in the `nodes` module.


Note that if the `removeInvisibleProtoNodes` functionality reamains in WbNode class, then you should also check the visibility of some functions (for example `WbNode::disconnectInternalNode`, `WbNode::removeFromFieldsOrParameters`, etc.) that can be made private.